Specs/MiddlewareLib/0.8.0/MiddlewareLib.podspec.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "MiddlewareLib",
+  "version": "0.8.0",
+  "summary": "Library that is the middleware between NPC´s Apps and NPC´s services",
+  "homepage": "https://github.com/indigotech/br-npc-middleware-lib-java/",
+  "license": {
+    "type": "Comercial"
+  },
+  "authors": {"Taqtile": "contato@taqtile.com"},
+  "source": {
+    "git": "https://github.com/indigotech/br-npc-middleware-lib-java/",
+    "tag": "0.8.0"
+  },
+  "platforms": {
+    "ios": "5.0"
+  },
+  "source_files": "ios_release/**/*.{h,m}",
+  "header_mappings_dir": "ios_release",
+  "libraries": [
+    "z",
+    "icucore"
+  ],
+  "frameworks": [
```

Comment:
@vnavarro 
Does your pod require really this framework or they are just requirements of J2ObjC?